skfuzzy/control/fuzzyvariable.py

```diff
@@ -11,6 +14,41 @@
     from .ordereddict import OrderedDict
 
 
+class FuzzyVariableAdjective(object):
```

Comment:
If I'm following, this class is intended to replace the (admittedly less than ideal) `self.active` structure for tracking the membership function of interest. I like the approach, but wonder if `Adjective` is the best name. It implies modification, where this doesn't really change anything - it just highlights a single membership function.

Something like "Active", "Operative," or "Emphasis" might be a bit more intuitively named. I'd lean toward "FuzzyVariableOperative"